src/nrniv/multisend.cpp

```diff
@@ -471,10 +471,12 @@ void nrn_multisend_receive(NrnThread* nt) {
     assert(nt == nrn_threads);
     TBUF
     double w2;
-    int ncons = 0;
```

Comment:
Feels like a steep price to pay for a warning, maybe `[[maybe_unused]]` is better?